airbyte-integrations/connectors/source-iterable/source_iterable/api.py

```diff
@@ -22,23 +22,33 @@
 # SOFTWARE.
 #
 
-
 import json
```

Comment:
what happen to permission of this file? 644 -> 755

Reply:
Fixed